src/Env.php

```diff
@@ -44,4 +46,54 @@ public static function get($name, $default = null)
             return $originalValue;
         }
     }
+    
+    /**
+     * 
+     * @param string $varname
+     * @return string
+     * @throws EnvVarNotFoundException
+     */
+    public static function requireEnv($varname)
+    {
+        $value = self::get($varname);
+        
+        if ($value === null) {
+            $message = 'Required environment variable: ' . $varname . ', not found.';
+            throw new EnvVarNotFoundException($message);
+        } elseif ($value === '') {
```

Comment:
Good catch.  I added a test expecting get() to return '', but it returned '   ' which failed.  I'll trim() whitespace from the value.